events/tests.py

```diff
@@ -64,3 +76,48 @@ def test_invalidation(title, date_time_start, date_time_end, expected_error):
     except ValidationError as error:
         current_error = error.messages[0]
     assert expected_error == current_error
+
+
+def create_event_participant(event, user, is_creator):
+    return EventParticipant(event_id=event, user_id=user, is_creator=is_creator)
+
+
+@pytest.fixture
+def event_participant_creator(new_event, user0):
+    return create_event_participant(new_event, user0, True)
+
+
+@pytest.fixture
+def event_participant_not_creator(new_event, user0):
+    return create_event_participant(new_event, user0, False)
+
+
+@pytest.fixture
+def persist_event_participant(event_participant_creator):
+    event_participant_creator.event_id.save()
+    event_participant_creator.user_id.save()
+    event_participant_creator.save()
+    return event_participant_creator
+
```

Comment:
please add a test to make sure we prevent adding the same user+ same event a few times

Reply:
Done